src/migrations/annotation.ts

```diff
@@ -0,0 +1,34 @@
+import { KustoQuery, defaultQuery } from 'types';
+
+type inputAnnotation = {
+  datasource: string;
+  database: string;
+  enable: boolean;
+  name: string;
+  iconColor: string;
+  query?: string;
+  target?: KustoQuery;
+  resultFormat?: string;
+};
+
+export const migrateAnnotation = (annotation: inputAnnotation) => {
+  if (annotation.target && annotation.target.database !== null) {
+    return annotation;
+  }
+
+  const newQuery: KustoQuery = {
+    ...defaultQuery,
+    ...(annotation.target ?? {}),
+    database: annotation.database,
+    query: annotation.query ?? '',
+    rawMode: true,
+    refId: annotation.target?.refId ?? 'Anno',
```

Comment:
Do you know where does 'Anno' come from? Does refId have to be unique? I'm not very familiar with this yet but I see elsewhere we do stuff like:
```
refId: `adx-${interpolatedQuery}`
```